mautic/mautic
# Prevent call page view event with mt()

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
We can call mt() multiple times.  At the moment Mautic support just send/pageview event.
Standard call with tracking code 

`mt('send', 'pageview');`

But you can call it more time on page.  If you call `mt();` Mautic send page hit, 
What is issue for me If we want use mt() for our own events. For example `mt('send', 'click');`


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  Add tracking code to to page
2.  Load page. Verify If tracking code is loaded properly
3. Then open browser console and call `mt('send', 'pageview');` . Page hit is fired again
4. Then  call `mt();` and page hit is fired again

![image](https://user-images.githubusercontent.com/462477/47534568-13a59180-d8b8-11e8-9fc6-e6525f72e107.png)


#### Steps to test this PR:
1.  Repeat all steps. 
2.  Just `mt('send', 'pageview');`  should call page hit in browser console
